tests/conftest.py

```diff
@@ -13,16 +11,13 @@ def vcr_config():
 
 @pytest.fixture
 def vcr_cassette_name(request):
-    """Name of the VCR cassette"""
-    f = request.function
-    name = request.node.name
-    mark = f.pytestmark[0]
-    args = mark.args
-    params = args[1]
-
-    if "table_name" in args[0]:
-        param = [param for param in params if param[0] in name][0]
-        table_name = param[0]
-        return table_name
-
-    return name
+    """Name of the VCR cassette, changed to {language}_{table_name}."""
+    test_arguments = request.node.callspec.params
```

Comment:
why did you need to change this? The way it should work, to my understanding, is that all requests are added to the same {table_name}.yaml. If you request the same table name with "de" and then with "en" these are different language parameters, so the two requests should be added to the same .yml file in my previous version. I would prefer this over twice the amount of yaml files. And it is actually better to have both versions in one file for comparison reasons

Reply:
Good point, I thought this would overwrite existing request. I will test this and will rollback the changes.

Reply:
Indeed this works as you suggested. I rolled back my changes to separate the cassettes by language.